extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -2071,13 +2071,13 @@ export class AmpA4A extends AMP.BaseElement {
     // Block synchronous XHR in ad. These are very rare, but super bad for UX
     // as they block the UI thread for the arbitrary amount of time until the
     // request completes.
-    let featurePolicies = 'sync-xhr=()';
+    let permissionPolicies = 'sync-xhr=();';
```

Comment:
Based on the spec it looks like the semicolon is only required if there's more than one directive, so please only add it in that case (like 2077) instead of here - once that's fixed I'll merge this PR!